docs/interfaces/proxy-ports.rst

```diff
@@ -0,0 +1,65 @@
+.. _proxy-ports:
+
+#######################
+ Exposing Custom Ports
+#######################
+
+Determined allows you to expose a custom network port in a task container, and access it using a
+local tunnel.
+
+For multi-container tasks, such as distributed training experiments, only the ports on the chief
+container (``rank=0``) will be exposed.
+
+***************
+ Configuration
+***************
+
+First, specify the ports in the ``environments -> proxy_ports`` section of the experiment or task
+config, for example:
+
+.. code:: yaml
+
+   environment:
+     proxy_ports:
+       - proxy_port: 8265
+         proxy_tcp: true
+
+Then launch your task or experiment normally. Once it's up, use the ``det`` CLI to start a tunnel.
```

Comment:
Launch your task or experiment normally. Than, use the ``det`` CLI to start a tunnel.